lightning/src/routing/gossip.rs

```diff
@@ -2140,9 +2222,38 @@ mod tests {
 			Err(e) => assert_eq!(e.err, "Already have chain-validated channel")
 		};
 
+		#[cfg(feature = "std")]
+		{
+			use std::time::Duration;
+			use util::time::tests::SinceEpoch as Instant;
```

Comment:
Could you use the name `SinceEpoch` instead of overriding? That way it will be easier to grep where it is used.